roles/example-cnf-app/tasks/retry-trex.yaml

```diff
@@ -37,6 +38,7 @@
       when:
```

Comment:
Is it good to add a condition check for a valid value in `trex_app_logs` ? since the errors are ignored, we don't need to execute this task when there is no content to write. I gree this task also ignores error, but its good to not invoice if the above task results in error.

Reply:
Hi @krsacme, thank you for checking, added the `stderr` check.